pkgs/development/web/pnpm-lock-export/update.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix wget nix-prefetch-github jq prefetch-npm-deps nodejs
+
+# shellcheck shell=bash
+
+if [ -n "$GITHUB_TOKEN" ]; then
+    TOKEN_ARGS=(--header "Authorization: token $GITHUB_TOKEN")
+fi
+
+if [[ $# -gt 1 || $1 == -* ]]; then
+    echo "Regenerates packaging data for the pnpm-lock-export packages."
+    echo "Usage: $0 [git release tag]"
+    exit 1
+fi
+
+set -x
```

Comment:
```suggestion
set -eou pipefail
```

Reply:
It's already there, a few lines down, to avoid the error on `version=$1`.